server/util/repeatingRide.ts

```diff
@@ -12,104 +11,45 @@ export default function initSchedule() {
   });
 }
 
-
 function createRepeatingRides() {
-  let tomorrowDate = moment(new Date());
-  tomorrowDate = tomorrowDate.add(1, 'days');
+  const tomorrowDate = moment().add(1, 'days');
   const tomorrowDateOnly = tomorrowDate.format('YYYY-MM-DD');
-
   const tomorrowDay = tomorrowDate.weekday();
 
   // condition: recurring ride that includes tomorrowDay
   const condition = new Condition()
     .where('recurring')
     .eq(true)
-    .and()
     .where('recurringDays')
-    .contains(tomorrowDay);
+    .contains(tomorrowDay)
+    .where('endDate')
+    .ge(tomorrowDateOnly)
+    .where('deleted')
+    .not()
+    .contains(tomorrowDateOnly);
```

Comment:
i think we still need to check that the startDate is less than tomorrow's date

Reply:
yep good catch